esmvaltool/cmorizers/data/datasets.yml

```diff
@@ -0,0 +1,884 @@
+# Dataset information
+---
+datasets:
+  APHRO-MA:
+    tier: 3
+    source: "http://aphrodite.st.hirosaki-u.ac.jp/download/"
+    last_access: 2020-03-06
+    info: |
+      Register at http://aphrodite.st.hirosaki-u.ac.jp/download/create/
+
+      To use the automatic download set your credentials in your .netrc file.
```

Comment:
Perhaps an example would be useful? I'm not if it should be placed here or in the doc.

Reply:
Added a link to an explanation on how .netrc files work